api/app/src/sequelize/migrations/20230104000000-create-org-facility.ts

```diff
@@ -0,0 +1,50 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import { Facility } from "../../models/medical/facility";
+import { Organization } from "../../models/medical/organization";
+import * as shared from "../migrations-shared";
+
+// Use 'Promise.all' when changes are independent of each other
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async transaction => {
```

Comment:
From what I can tell Postgres doesn't create indexes on foreign keys, so we should create one on `Facility` for the combination of `organizationId, systemRootOid` - if we keep `systemRootOid` as part of the PK/Unique contraint.

Reply:
Same idea for the Organization table.